src/scales/scale.time.js

```diff
@@ -36,6 +36,19 @@ module.exports = function(Chart) {
 		},
 		ticks: {
 			autoSkip: false
+		},
+		majorTicks: {
```

Comment:
I mean, global options for `ticks.minor` and `ticks.major` would be empty objects:
```javascript
ticks: {
  minor: {}, //< use common tick options
  major: {}  //< use common tick options
}
```

and we keep current [global `ticks` options](/chartjs/Chart.js/blob/master/src/core/core.scale.js#L39) so when computing effective minor and major options, it would fallback to `ticks` global options:

* (effective) minor options = `deep merge(ticks, tick.minor)`
* (effective) major options = `deep merge(ticks, tick.major)`

It's not a breaking change because minor and major ticks would use the `ticks` values by default, same as what we have in 2.6. Also having the shorthand `ticks.minor: false` (same for major) to hide minor or major ticks would be great.